config/labels.yml

```diff
@@ -0,0 +1,51 @@
+release_labels: &release_labels
```

Comment:
It creates an anchor so this hash can be merged into other hashes with the `<<` operator below it